dataprofiler/tests/labelers/test_data_processing.py

```diff
@@ -3017,9 +3017,8 @@ def test_process(self):
             true_positive_dict=params["true_positive_dict"],
             positive_threshold_config=params["positive_threshold_config"],
         )
-        process_output = processor.process(data)
-
-        self.assertEqual(process_output, expected_output)
+        with self.assertRaises(TypeError):
+            process_output = processor.process(data)
```

Comment:
this raises `TypeError` when just testing the post processor becuase the `process()` label parameter is None and `NoneType` is not subscriptable. We could do a more elegant error handling here

Reply:
out dated